### PR TITLE
Make param-dump display current values

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Should there be a `--with-values` option?

**Comment by danielbachhuber** [comment]
I'm not convinced this is the right direction. I need to give it more thought.

**Comment by danielbachhuber** [comment]
@szepeviktor What was your use-case for this again?

**Comment by szepeviktor** [comment]
When I am debugging wp-cli I finish faster.

File: php/commands/cli.php [hunk 1]
```diff
@@ -70,12 +70,34 @@ function info( $_, $assoc_args ) {
 	}
 
 	/**
-	 * Dump the list of global parameters, as JSON.
+	 * Dump the list of global parameters, as JSON or by var_export.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--format=<format>]
+	 * : Accepted values: var_export
```
Comment (danielbachhuber) [inline php/commands/cli.php]: Also accepts `json`

**Comment by danielbachhuber** [comment]
Can you merge master to refresh the branch?

**Comment by szepeviktor** [comment]
@danielbachhuber Could you finish this, I don't have time. Sorry.

**Comment by szepeviktor** [comment]
Here it is: https://github.com/szepeviktor/wp-cli/commit/97a0d042ecd7fc7afc2951e452e3c9bdb8b94467

**Comment by szepeviktor** [comment]
I've just noticed the `cli` command has no test, excluding `update`.

**Comment by szepeviktor** [comment]
@danielbachhuber Please merge.

**Comment by szepeviktor** [comment]
Sorry for the mess. *Still learning git...*

**Comment by szepeviktor** [comment]
Should I go th new way?
```php
if ( \WP_CLI\Utils\get_flag_value( $assoc_args, 'with-values' ) ) {
```

**Comment by danielbachhuber** [comment]
> Should I go the new way?

Yes please

**Comment by szepeviktor** [comment]
Done.

**Pull request merged**